src/2015-site-policies.rst

```diff
@@ -0,0 +1,81 @@
+Posting and commenting policy - Oct 2015
+########################################
+
+:author: C\. Titus Brown
+:tags: policy
+:date: 2015-10-19
+:slug: 2015-site-policies
+:category: misc
+
+This version of the site policy was posted in Oct 2015 and applies
+going forward from then.  I'll provide an updated link whenever I
+update the policy.
+
+The text was lifted from Captain Awkward's site policy at the URL
+http://captainawkward.com/site-policies-and-faqs/ on Oct 19, 2015, and
+then modified (significantly) from there.
+
+I'm open to suggestions for additions and removals; please post them
+in comments.  However, this set of policies stands until I change them -
+thanks for abiding by them!
+
+Copyright Notice
+----------------
+
+I own the copyright to all of my writing. Guest bloggers retain their
+own copyright. Quote all of us freely!  Link & attribute always!
+
+Questions
+---------
+
+By posting your comments to the blog you are granting me the right, in
+perpetuity, to use, quote, alter, and/or display those questions and
```

Comment:
Possible reference to the "fart fart fart" comment moderation style? http://sarah.thesharps.us/2015/10/05/closing-a-door/#comment-147828